src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,34 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, $cardWidth);
+  justify-content: center;
+  gap: 48px;
+
+  max-width: 944px;
+  margin: 0 auto;
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: 200px;
```

Comment:
You write 'width' in 'cardWidth'. I think you can delete this string or write a variable that you created.